packages/components/inputs/money-input/src/money-input.tsx

```diff
@@ -247,7 +247,7 @@ export const parseRawAmountToNumber = (rawAmount: string, locale: string) => {
 //
 // This function expects the "amount" to be a trimmed value.
 
-type TCurrencyCode = keyof typeof currencies;
+export type TCurrencyCode = keyof typeof currencies;
```

Comment:
Bear in mind this change is also done in [this PR](https://github.com/commercetools/ui-kit/pull/2099) from @Rhotimee although he used a different approach to expose the types.

Reply:
Oh my, I was supposed to take it out after I checked in with him. I will take out the change from my end.